changelog

```diff
@@ -1,3 +1,193 @@
+          * Allow the player to decline the follow-up mission.
+        * Southern Fiance:
+          * Reduced the possible source planets to urban core Syndicate worlds.
+          * Cut all of the follow up missions except for the marriage letter. The only mission string that get multiple epilogue missions is the Free Worlds campaign; having this many follow-ups to a single mission is intrusive at best.
+          * Moved some of the information from the later missions up into the first mission to compensate.
+      * "Deep: Interrogation" no longer offers during the main campaign. (@Saugia)
+      * Various improvements to some pirate jobs. (@ziproot)
+        * "Drug Running" missions are now "infiltrating";
+        * Added (and used) generic aborted phrases for pirate slave jobs;
+        * Failing or aborting a pirate slave job causes a reduction in reputation with the "Pirate" government;
+        * Many jobs now have a minimum travel distance of 4, instead of 3, to prevent them from involving travelling between the Men system and a CCOR world, as this route does not involve any territory hostile to pirates. Affected jobs:
+          * "Cargo Smuggling", "Bulk", and "Stealth (South)" variations,
+          * "Drug Running", and "Stealth (South)" variation,
+          * "Slave Transport", and "Bulk" variation,
+          * "Escort Illegal Cargo" South variation, and
+          * "Escort Stolen Vessel" South variation;
+        * The targets NPCs in "Eliminating Law Enforcement" jobs have been given the "marked" personality;
+        * Alphabetized the list of personalities for the target NPCs in "Eliminating Competition" jobs;
+        * Various jobs will only place NPCs in systems neighbored by a non-Pirate system. Affected jobs are:
+          * "Eliminating Law Enforcement" Core and South variations,
+          * "Raid on Merchants" North and South variations, and
+          * "Cargo Theft (South)"; and
+        * Various jobs will only offer from systems neighbored by a non-Pirate system (that is, they will no longer offer from CCOR worlds). Affected jobs are:
+          * "Eliminating Law Enforcement (South)",
+          * "Southern Pirate Defense",
+          * "Cargo Theft (South)", and
+          * "FW Assassination".
+      * Mark Betelgeuse in "Deep Scientist Rescue 0" instead of making it a waypoint. (@roadrunner56)
+      * Use marks instead of waypoints in "Pact Recon 3". (@ziproot)
+    * Balance:
+      * Reduce the chances of finding Bactrians in random pirate fleet spawns by half. (@alexrovw)
+      * Nerf the Unfettered medical jobs. (@Hurleveur)
+      * Cut Wanderer sun reactor energy capacity in half + -20% energy generation on the biochemical cells. (@Quantumshark)
+      * Cut the asteroid scanner's price in half. (@AvianGeneticist)
+      * Revert old human missile initial velocity changes + additional tweaks. (@Anarchist2)
```

Comment:
```suggestion
      * Revert old human missile initial velocity changes (@Anarchist2)
      * Tweak the capacity of some human missile pods (@Anarchist2)
        * Javelin mini pod 40 -> 30
        * Sidewinder pod 4 -> 6
        * Typhoon pod 2 -> 3
```
Again, splitting this by content rather than by PR, plus specifying the changes rather than just saying "additional tweaks".